com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader

```diff
@@ -2,6 +2,7 @@ Shader "Hidden/Universal Render Pipeline/UberPost"
 {
     HLSLINCLUDE
         #pragma exclude_renderers gles
+        #pragma multi_compile_local_fragment _ _GAMMA_20
```

Comment:
`_GAMMA_20` and `_LINEAR_TO_SRGB_CONVERSION` are mutually exclusive right? If so, I guess we can make 
`multi_compile_local_fragment _ GAMMA_20 _LINEAR_TO_SRGB_CONVERSION` and save some variants.

Reply:
I dug into this a bit and realized that FSR didn't play well with the gamma color encoding project setting so I fixed that in a new commit and addressed the unnecessary shader variants as well.